services/blockchain-connector/shared/sdk/client.js

```diff
@@ -40,20 +40,18 @@ let isInstantiating = false;
 const checkIsClientAlive = () => clientCache && clientCache._channel.isAlive;
 
 // eslint-disable-next-line consistent-return
-const instantiateClient = async () => {
+const instantiateClient = async (isForceUpdate = false) => {
```

Comment:
We may not need this incase we go with the other suggestion

```suggestion
const instantiateClient = async () => {
```